project/docs/Makefile

```diff
@@ -0,0 +1,153 @@
+# Makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line.
+SPHINXOPTS    =
+SPHINXBUILD   = sphinx-build
```

Comment:
@kunicmarko20 can you add the env var usage here as i did in https://github.com/sonata-project/SonataSeoBundle/pull/282 pls doing so we can locally use a separate environment, which is common in python projects